### PR TITLE
Style home

### PR DESCRIPTION
**Pull request by jonperrelle**

### CODE REVIEW

File: browser/js/home/home.html [hunk 1]
```diff
@@ -1,33 +1,42 @@
...
+            <form name="form" class="upload-form">
+              <button class="btn select-btn" ng-show="!file" ngf-select ng-model="file">Select file to upload</button>
+              <p class="file-upload" ng-show="file">{{file.name}}</p>
+              <button ng-show="file" class="btn btn-danger upload-btn" type="submit" ng-click="file=false">Cancel</button>
+              <button ng-show="file" class="btn btn-primary upload-btn" type="submit" ng-click="uploadFile()">Upload</button>
+            </form>
+          </div>
+        </div>
       </div>
+      <div class="col-xs-1"></div>
```
Comment (geoffbass) [inline browser/js/home/home.html]: what's this guy for?

File: browser/scss/home/main.scss [hunk 2]
```diff
@@ -13,18 +13,70 @@
...
-	margin-top: 30px;
+  margin-top: 70px;
+}
+
+.graph-home {
+  margin: 50px 0;
+  background-image: url('graph-background.png');
+  background-repeat: no-repeat;
+  background-size: 100% 100%;
+  height: 500px;
```
Comment (geoffbass) [inline browser/scss/home/main.scss]: Are we going to have hard-coded pixel values in the final product or will everything be relative to screen size / resizable?

**Comment by jonperrelle** [comment]
@cde-v

File: browser/js/common/directives/dataset-table/dataset-table.directive.html [hunk 3]
```diff
@@ -0,0 +1,24 @@
...
+      </thead>
+      <tbody>
+        <tr ng-repeat="row in $data">
+          <td ng-repeat="column in columns">
+            {{row[column]}}
+          </td>
+        </tr>
+      </tbody>
+    </table>
+    <bar-chart data="rows" category="category" metric="metric"></bar-chart>
```
Comment (geoffbass) [inline browser/js/common/directives/dataset-table/dataset-table.directive.html]: don't think the bar chart should be attached to this on the table directive

**Pull request merged**